docs/docs/modeling.md

```diff
@@ -112,6 +113,39 @@ classDiagram
 
 ## Tenant - テナント
 
+### Tenant ID (Identifier)
+
+The Tenant ID is the UUID.
+
+### Name
+
+The Tenant has a name.
+
+Constraints:
+
+- Name must be at least one character no more than 15 characters.
+- Name must be unique (no duplicates).
+- Name can set a group name as they like, not limited to the actual personal name.
+- Name can change anytime.
```

Comment:
IMO, I think it's better to have the same format as `Owner` since the way you wrote constraints is different from it.
https://github.com/Tascurator/tascurator-documents/pull/28#discussion_r1477183777

Reply:
Okay, I have revised the format of the Owner because this format is more versatile.